vignettes/WQPDataHarmonization.Rmd

```diff
@@ -85,7 +85,7 @@ CharacteristicName
 ActivityMediaName
 endDate
 
-Tip 1) All the query filters for the WQP work as an AND but within the fields there are ORs. So for example, characteristics – if you choose pH & DO – it’s an OR. Similarly, if you choose VA and IL, it’s an OR. But the combo of fields are ANDs, such as State/VA AND Characteristic/DO". "Characteristic" and "Characteristic Type" on the WQP also work as an AND.
+Tip 1) All the query filters for the WQP work as an AND but within the fields there are ORs. So for example, characteristics - if you choose pH & DO - it's an OR. Similarly, if you choose VA and IL, it's an OR. But the combo of fields are ANDs, such as State/VA AND Characteristic/DO". "Characteristic" and "Characteristic Type" on the WQP also work as an AND.
```

Comment:
There were some non-ASCII characters in here. That usually happens when copy/pasting from Word.